src/modules/packages.c

```diff
@@ -53,6 +53,34 @@ static uint32_t getNumStrings(const char* filename, const char* needle)
     return count;
 }
 
+static void getActiveBranch()
```

Comment:
The method name should contain Manjaro to make clear it is a manjaro only function.